src/util/serializing/BinObjectEncoding.cpp

```diff
@@ -1,11 +1,11 @@
 #include "util/serializing/BinObjectEncoding.h"
 
-#include <glib.h>  // for g_string_append_len
+#include <string>
 
 BinObjectEncoding::BinObjectEncoding() = default;
 
 BinObjectEncoding::~BinObjectEncoding() = default;
 
 void BinObjectEncoding::addData(const void* data, int len) {
-    g_string_append_len(this->data, static_cast<const char*>(data), len);
+    this->data.append(std::string((const char*)data, (const char*)data + len));
```

Comment:
use std::stringview instead of creating/copiing a string, which is destructed immediatelly.

Reply:
`this->data` is now a `std::vector<std::byte>`, so this is done different now